src/test/java/jgfx/javagradlefx/SaveMapTest.java

```diff
@@ -0,0 +1,42 @@
+package jgfx.javagradlefx;
+
+import app.controller.Settings;
+import app.model.map.Map;
+import app.model.map.SaveMap;
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import static java.nio.file.Files.deleteIfExists;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class SaveMapTest {
+
+    @Test
+    void CreateFile()
+    {
+        Settings settings = new Settings();
+        settings.setName("createFileTest");
+        Map map = new Map(settings);
+
+        SaveMap safeMap = new SaveMap();
+        safeMap.saveMap(map);
+
+        String filePathName = "src/main/java/app/model/map/files/createFileTest.txt";
+
+        try
+        {
+            Scanner scanner = new Scanner(filePathName);
+
+            Path path = Paths.get(filePathName);
+            deleteIfExists(path);
```

Comment:
When testing files, It's good practice to write a `@BeforeAll` and `@AfterAll` method that does your build and breakdown for all the tests, then you only have to do one of these deletes in the file